timemachine/cpp/src/barostat.hpp

```diff
@@ -56,13 +56,11 @@ class MonteCarloBarostat {
     int *d_num_attempted_;
     int *d_num_accepted_;
 
-    unsigned long long *d_u_buffer_;
-    unsigned long long *d_u_after_buffer_;
+    __int128 *d_u_buffer_;
+    __int128 *d_u_after_buffer_;
```

Comment:
That would make sense, as I know @proteneer and I had talked about 96 bit instead of 128.

If we want to do this, I would prefer doing it in a separate pass as this has gotten pretty large already.